react/react2/week1/homework/src/App.js

```diff
@@ -1,7 +1,10 @@
 import './App.css';
-import React, {useCallback, useEffect, useState} from 'react';
+import React, { useCallback, useEffect, useState, createContext } from 'react';
+import H1Title from './H1Title'
 
-function App() {
+export const AppContext = createContext(null)
+
+export default function App() {
   const [loading, setLoading] = useState(true)
   const [data, setData] = useState("")
   const [user, setUser] = useState("SomayehDvr")
```

Comment:
You don't need this. First data fetch should happen only after input. In your case, once the page is loaded, input is already filled. It ruinss the purpose of a placeholder. You should include check in the lifecycle, if there is something in a query, and the perform fetch with that query, else "no data component should be shown"